src/program/transient/transient.lua

```diff
@@ -0,0 +1,205 @@
+module(..., package.seeall)
+
+local engine    = require("core.app")
+local counter   = require("core.counter")
+local config    = require("core.config")
+local timer     = require("core.timer")
+local pci       = require("lib.hardware.pci")
+local Intel82599= require("apps.intel.intel_app").Intel82599
+local basic_apps = require("apps.basic.basic_apps")
+local main      = require("core.main")
+local PcapReader= require("apps.pcap.pcap").PcapReader
+local lib = require("core.lib")
+local ffi = require("ffi")
+local C = ffi.C
```

Comment:
`C` is not used. The headers seem to be column-aligned but some are not. It's not a requirement of SnS to column-align headers but it's strange to see it a new file.